README

```diff
@@ -1,26 +1,36 @@
-Модуль ngx_http_requestid_module предназначен для генерации уникального идентификатора запроса.
+The Ngx_http_requestid_module module is designed to generate a unique request ID.
 
-Результатом работы модуля является переменная nginx $request_id - уникальный идентификатор запроса.
+The result of the module is a variable $request_id containing an unique request ID.
 
-Идентификатор представляет собой md5('ngx_pid,connection_number,rand()), где:
+The ID has following structure: (time){13} + (md5(ngx_pid, connection_number, rand())){19}
+  * time - Timestamp in milliseconds
+  * Ngx_pid - Nginx PID process
+  * Connection_number - COUNTER current connection instance nginx
+  * Rand () - random number
 
- * ngx_pid - PID процесса nginx
- * connection_number - текущий COUNTER соединений инстанса nginx
- * rand() - случайное число 
+Supports both old and new configuration methods including dynamic compilation.
 
-Пример конфигурации:
+Example configurations:
 
+# Set Proxy Header
 location / {
-    requestid on;
-    proxy_pass http://127.0.0.1;
-    proxy_set_header X-Request-Id $request_id
+     requestid on;
+     proxy_pass http://127.0.0.1;
+     proxy_set_header X-Request-Id $ request_id
 }
 
-Конфигурация:
 
-syntax: requestid on|off
+# Set HTTP Header
+location / {
+     add_header X-Request-Id $request_id;
+}
+
+
+Configuration:
+
+syntax: requestid on | off
 default: off
 context: server, location
 
-Директива активирует работу модуля. 
-Если модуль включен, в переменную $request_id записывается идентификатор запроса. Переменную можно записывать в access_log, используя директиву log_format.
\ No newline at end of file
+If the module is enabled the $request_id variable can be written in the access_log, using the directive log_format.
```

Comment:
When the module is enabled with 'requestid on', use $request_id with log_format directive to add request ID to access log.